<launch>

  <!-- Teo teleop -->
  <node pkg="teo_teleop"
        type="teleop_teo_iri_wiimote"
        name="teleop_teo"
        output="screen" >
    <remap from="/teo/command_velocity" to="/WiiTwist" />
  </node>

  <!-- ros wiimote node -->
  <node respawn="true"
        pkg="wiimote"
        type="wiimote_node.py"
        name="joy"
        >
  </node>

</launch>
